addon/templates/components/polaris-text-field.hbs

```diff
@@ -1,121 +1,104 @@
-      onblur={{action "handleBlur"}}
-      onclick={{action "handleClick"}}
+      class={{this.textFieldClasses}}
+      {{on "focus" this.handleFocus}}
+      {{on "blur" this.handleBlur}}
+      {{on "click" this.handleClick}}
     >
-
-      {{#if prefix}}
-        <div data-test-text-field-prefix class="Polaris-TextField__Prefix" id="{{id}}Prefix">
-          {{render-content prefix}}
+      {{#if @prefix}}
+        <div data-test-text-field-prefix class="Polaris-TextField__Prefix" id="{{this.id}}Prefix">
+          {{render-content @prefix}}
         </div>
       {{/if}}
 
-      {{! template-lint-disable attribute-indentation }}
-      {{!-- TODO remove rule above once linter handles this correctly --}}
-      {{#with
-        (component "wrapper-element"
-          data-test-text-field-input=true
-          name=name
-          id=id
-          disabled=disabled
-          readonly=readOnly
-          role=role
-          autofocus=autoFocus
-          placeholder=placeholder
-          autocomplete=autoCompleteInputs
-          max=max
-          min=min
-          step=step
-          minlength=minLength
-          maxlength=maxLength
-          spellcheck=spellCheck
-          value=normalizedValue
-          class=inputClassName
-          aria-describedby=ariaDescribedBy
-          aria-label=label
-          aria-labelledby=ariaLabelledBy
-          aria-invalid=ariaInvalid
-          aria-owns=ariaOwns
-          aria-activedescendant=ariaActiveDescendant
-          aria-autocomplete=ariaAutocomplete
-          aria-controls=ariaControls
-          oninput=(action "handleChange")
-          onfocus=(action onFocus)
-          onblur=(action onBlur)
-          onkeypress=(action "handleKeyPress")
-        )
-        as |inputElement|
-      }}
-        {{#if multiline}}
-          {{component
-            inputElement
-            tagName="textarea"
-            aria-multiline=true
-            style=heightStyle
-          }}
-        {{else}}
-          {{component
-            inputElement
-            tagName="input"
-            type=inputType
-          }}
-        {{/if}}
-      {{/with}}
+      {{#let (element (if @multiline "textarea" "input")) as |Input|}}
+        <Input
+          data-test-text-field-input
+          type={{unless @multiline this.inputType}}
+          style={{if @multiline this.heightStyle}}
+          name={{@name}}
+          id={{this.id}}
+          disabled={{@disabled}}
+          readonly={{@readOnly}}
+          role={{@role}}
+          autofocus={{@autoFocus}}
+          placeholder={{@placeholder}}
+          autocomplete={{this.autoCompleteInputs}}
+          max={{@max}}
+          min={{@min}}
+          step={{@step}}
+          minlength={{@minLength}}
+          maxlength={{@maxLength}}
+          spellcheck={{@spellCheck}}
+          value={{this.normalizedValue}}
+          class={{this.inputClassName}}
+          aria-multiline={{@multiline}}
```

Comment:
Does this add `aria-multiline="false"` or anything similar if `@multiline={{true}}` is passed to the `PolarisTextField`? If so, can we check if that replicates the React implementation and if not, update it? 🙇

Reply:
made sure it matches react implementation